SwaggerDemo/src/main/java/com/globant/bootcamp/swagger2/controller/Swagger2DemoController.java

```diff
@@ -0,0 +1,65 @@
+package com.globant.bootcamp.swagger2.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.globant.bootcamp.swagger2.model.Glober;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@Api(value = "Swagger2DemoController")
+@RestController
+public class Swagger2DemoController {
+
+	private List<Glober> globers = new ArrayList<>();

```

Comment:
Better to put a Thread Safe Collections

Reply:
Que les quede de tarea de a los chicos 
entender por qué hay riesgo de usar listas :P